tools/test/scripts/admin_tools/net-changes.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# list all network changes submitted through the northbound gNMI interface run'
+
+ONOS_CONFIG_ROOT=$HOME/go/src/github.com/onosproject/onos-config
+
+go run github.com/onosproject/onos-config/cmd/admin/net-changes \
```

Comment:
With this and all the other utils there is now no need to give keyPath and certPath - see updates in run.md. 
Not much left to script here

Reply:
@SeanCondon @Andrea-Campanella 
I agree with what you said and I think with 
new changes no scripts will be needed  
for internal commands. In my opinion, in the long term if we don’t 
want to rely on gnmi cli, we need to equip the onos-config to 
a real CLI (not just a binary which accepts arguments) that we can 
execute different commands on top of NB. I think we can close this pull request.

Reply:
> @SeanCondon @Andrea-Campanella
> I agree with what you said and I think with
> new changes no scripts will be needed
> for internal commands. In my opinion, in the long term if we don’t
> want to rely on gnmi cli, we need to equip the onos-config to
> a real CLI (not just a binary which accepts arguments) that we can
> execute different commands on top of NB. I think we can close this pull request.

Thanks Adib. I think this is the best for the moment. Even with "go run .../onos-config-manager" the list of stores will soon disappear as they will be transitioned to Atomix